assets/src/js/block-button.js

```diff
@@ -0,0 +1,58 @@
+const { registerBlockType } = wp.blocks;
```

Comment:
This whole file lacks documentation
Ref: 
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/
https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/